src/SeqCli/Cli/Features/BatchSizeFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SeqCli.Util;
```

Comment:
Missing license header here

Reply:
:+1: fine to add this in the next round of updates.